Fusion security templates not compatible.

After `Use Lit 2.0.0-rc.1 #10023` the `test-spring-security-fusion` module fails to build due to usage of `internalProperty`
in view templates

```
[02:32:01]	[Step 2/5] [ERROR] WARNING in ./views/public/public-view.ts 36:4-20
[02:32:01]	[Step 2/5] [ERROR] "export 'internalProperty' was not found in 'lit-element'
```

--- a/vaadin-spring-tests/test-spring-security-fusion/frontend/views/admin/admin-view.ts
+++ b/vaadin-spring-tests/test-spring-security-fusion/frontend/views/admin/admin-view.ts
@@ -2,17 +2,12 @@ import "@vaadin/vaadin-button";
 import "@vaadin/vaadin-text-field";
 import { BalanceEndpoint } from "Frontend/generated/BalanceEndpoint";
 import { appStore } from "Frontend/stores/app-store";
-import {
-  customElement,
-  html,
-  internalProperty,
-  PropertyValues,
-} from "lit-element";
+import { customElement, html, state, PropertyValues } from "lit-element";
 import { View } from "../view";
 
 @customElement("admin-view")
 export class AdminTSView extends View {
-  @internalProperty()
+  @state()
   private balance: number = 0;
 
   render() {

--- a/vaadin-spring-tests/test-spring-security-fusion/frontend/views/login-view.ts
+++ b/vaadin-spring-tests/test-spring-security-fusion/frontend/views/login-view.ts
@@ -1,13 +1,13 @@
 import { LoginResult } from "@vaadin/flow-frontend";
 import { AfterEnterObserver, RouterLocation } from "@vaadin/router";
 import "@vaadin/vaadin-login/vaadin-login-overlay";
-import { customElement, html, internalProperty } from "lit-element";
+import { customElement, html, state } from "lit-element";
 import { login } from "../auth";
 import { View } from "./view";
 
 @customElement("login-view")
 export class LoginView extends View implements AfterEnterObserver {
-  @internalProperty()
+  @state()
   private error = false;
 
   // the url to redirect to after a successful login

--- a/vaadin-spring-tests/test-spring-security-fusion/frontend/views/private/private-view.ts
+++ b/vaadin-spring-tests/test-spring-security-fusion/frontend/views/private/private-view.ts
@@ -2,17 +2,12 @@ import "@vaadin/vaadin-button";
 import "@vaadin/vaadin-text-field";
 import { BalanceEndpoint } from "Frontend/generated/BalanceEndpoint";
 import { appStore } from "Frontend/stores/app-store";
-import {
-  customElement,
-  html,
-  internalProperty,
-  PropertyValues,
-} from "lit-element";
+import { customElement, html, state, PropertyValues } from "lit-element";
 import { View } from "../view";
 
 @customElement("private-view")
 export class PrivateTSView extends View {
-  @internalProperty()
+  @state()
   private balance: number = 0;
 
   render() {

--- a/vaadin-spring-tests/test-spring-security-fusion/frontend/views/public/public-view.ts
+++ b/vaadin-spring-tests/test-spring-security-fusion/frontend/views/public/public-view.ts
@@ -1,12 +1,12 @@
 import "@vaadin/vaadin-button";
 import "@vaadin/vaadin-text-field";
 import { PublicEndpoint } from "Frontend/generated/PublicEndpoint";
-import { customElement, html, internalProperty } from "lit-element";
+import { customElement, html, state } from "lit-element";
 import { View } from "../view";
 
 @customElement("public-view")
 export class PublicTSView extends View {
-  @internalProperty()
+  @state()
   private time: string = "";
 
   async connectedCallback() {

